doc/tutorials/rdt_configuration.rst

```diff
@@ -257,10 +257,10 @@ Configure RDT for VM Using VM Configuration
 
    .. code-block:: none
 
-      $ make hypervisor BOARD=apl-up2 FIRMWARE=uefi
+      $ make hypervisor BOARD=tgl-rvp FIRMWARE=uefi
       ...
 
-      # these operations are done on UP2 board
+      # these operations are done on tgl-rvp board
```

Comment:
We need a deeper look at that doc, do we use an eMMC storage device for example on this `tgl-rvp` board?

The doc also talks about `acrn.efi` which we need to update.

Reply:
Updated.